origin.spec

```diff
@@ -289,6 +289,8 @@ done
 install -d -m 755 %{buildroot}%{_sysconfdir}/systemd/system.conf.d/
 install -p -m 644 contrib/systemd/origin-accounting.conf %{buildroot}%{_sysconfdir}/systemd/system.conf.d/
 
+%clean
```

Comment:
Hilariously, doing `--no-cleanup` on `tito`, `--noclean` for `rpmbuild` *and* this empty `%clean` I *still* see cleanup happening. Still trying to figure out how and why ...